cypress/integration/tests/administrator/proxy/proxy.enable.disable.test.ts

```diff
@@ -22,6 +22,27 @@ describe("Proxy operations", () => {
         preservecookies();
     });
 
+    it("Http Proxy validation", function () {
```

Comment:
Also rename the test to "Proxy port and host field validation"

Reply:
Solved in a9c764a5336d595c30dd97b8ee17ad01872b0ef1.